packages/gatsby/src/gatsby-node.js

```diff
@@ -0,0 +1,24 @@
+exports.onCreateWebpackConfig = ({ plugins, actions }) => {
+  actions.setWebpackConfig({
+    plugins: [
+      plugins.define({
+        __SENTRY_RELEASE__: JSON.stringify(
+          // GitHub Actions - https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables
+          process.env.GITHUB_SHA ||
+            // Netlify - https://docs.netlify.com/configure-builds/environment-variables/#build-metadata
+            process.env.COMMIT_REF ||
+            // Vercel - https://vercel.com/docs/v2/build-step#system-environment-variables
+            process.env.VERCEL_GITHUB_COMMIT_SHA ||
+            process.env.VERCEL_GITLAB_COMMIT_SHA ||
+            process.env.VERCEL_BITBUCKET_COMMIT_SHA ||
+            // Zeit (now known as Vercel)
+            process.env.ZEIT_GITHUB_COMMIT_SHA ||
+            process.env.ZEIT_GITLAB_COMMIT_SHA ||
+            process.env.ZEIT_BITBUCKET_COMMIT_SHA ||
+            '',
```

Comment:
Is there no way for us to make this a user set option without relying on environmental variables? Can we provide some kind of `pluginOptions` like they can with `onClientEntry` and `pluginParams`?

Reply:
this is still able to be overwritten from pluginOptions, but we should _not_ make a user set anything unless they absolutely have to. frictionless has always been the goal.